src/ManagedServices/ManagedServices/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+
+## Version 0.0.2
```

Comment:
This should be 1.0.0 - they are going GA this release.  This will also mean including them in Az.

Reply:
Let me know if the commit I just pushed covers everything required to add it to Az.